interfaces/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@reshuffle/interfaces",
-  "version": "0.5.0",
+  "version": "0.5.1",
```

Comment:
Need to replace "0.5.0" with "0.5.1" in all files due to rush<->concord hostility.